### ISSUE TITLE
Invalid Command 

### ISSUE THREAD

**Issue by alidcastano** [jekyll/jekyll-compose#35]
I am unable to get Jekyll compose to work. At first, I got it to work after placing the Ruby installation directory in my $PATH variable, as discussed [here](http://stackoverflow.com/questions/19276621/jekyll-installed-but-command-not-found). But then, regretfully, I did bundle install and it stopped working. I don't really know how to get it to work now. 

This is my gemfile:
```
source 'https://rubygems.org'

gem 'jekyll'
gem 'github-pages'

group :jekyll_plugins do
  gem 'jekyll-compose'
  gem 'jekyll-paginate'
end
```

This is my gem env:
```
RubyGems Environment:
  - RUBYGEMS VERSION: 2.5.1
  - RUBY VERSION: 2.0.0 (2014-05-08 patchlevel 481) [universal.x86_64-darwin14]
  - INSTALLATION DIRECTORY: /Library/Ruby/Gems/2.0.0
  - USER INSTALLATION DIRECTORY: /Users/acastano/.gem/ruby/2.0.0
  - RUBY EXECUTABLE: /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
  - EXECUTABLE DIRECTORY: /usr/bin
  - SPEC CACHE DIRECTORY: /Users/acastano/.gem/specs
  - SYSTEM CONFIGURATION DIRECTORY: /Library/Ruby/Site
  - RUBYGEMS PLATFORMS:
    - ruby
    - universal-darwin-14
  - GEM PATHS:
     - /Library/Ruby/Gems/2.0.0
     - /Users/acastano/.gem/ruby/2.0.0
     - /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/gems/2.0.0
  - GEM CONFIGURATION:
     - :update_sources => true
     - :verbose => true
     - :backtrace => false
     - :bulk_threshold => 1000
  - REMOTE SOURCES:
     - https://rubygems.org/
  - SHELL PATH:
     - /Library/Frameworks/Python.framework/Versions/2.7/bin
     - /Library/Frameworks/Python.framework/Versions/Current/bin
     - /usr/local/bin
     - /usr/bin
     - /bin
     - /usr/sbin
     - /sbin
     - /opt/X11/bin
     - /usr/local/git/bin
     - /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
     - /Library/Ruby/Gems/2.0.0/gems/
     - /Users/acastano/.gem/ruby/2.0.0
```

Are my $PATH configurations correct? Could there be a conflict between my gemfile and my gemfile.lock now? 

I'm unsure of how to proceed. The basic jekyll commands still work, but I would really like to install this plugin as well. Thanks in advance for the help

**Comment by parkr** [comment]
Your problem is here:

```ruby
gem 'github-pages'
```

The `github-pages` gem is not compatible with jekyll-compose, because it does not have a recent enough version of Jekyll. See https://github.com/github/pages-gem/pull/175 for the upgrade status of `github-pages` and the README of this repo for its constraint on Jekyll version. I can add a constraint to the jekylll-compose ge, too.

**Comment by parkr** [comment]
Just released v0.4.1 which will automatically yell at you if you don't have v2.5.0 or greater.
